### PR TITLE
fix: inverted and outline button contrast

### PR DESCRIPTION
**Pull request by jordmccord**

### CODE REVIEW

File: packages/native-ui/src/config/theme/helpers/Button.ts [hunk 1]
```diff
@@ -8,50 +8,53 @@ export const invertedOutlineScheme = (scheme: Scheme) => ({
...
       },
     },
 
     ':disabled': {
-      borderColor: `$${scheme}700`,
+      borderColor: `$${scheme}600`,
 
       _text: {
-        color: `$${scheme}700`,
+        color: `$${scheme}600`,
```
Comment (noeliacabane) [inline packages/native-ui/src/config/theme/helpers/Button.ts:50]: question here: does this apply to ghost too?

Comment (noeliacabane) [reply packages/native-ui/src/config/theme/helpers/Button.ts:50]: Screenshot 2024-08-08 at 16 06 39


the `disabled` on `inverted` Button `variant="ghost"` was changed from 700 to 600

Screenshot 2024-08-08 at 16 09 09

File: packages/native-ui/src/config/theme/helpers/Button.ts [hunk 2]
```diff
@@ -8,50 +8,53 @@ export const invertedOutlineScheme = (scheme: Scheme) => ({
...
     },
 
     _spinner: {
       _svg: {
-        color: `$${scheme}50`,
+        color: `$${scheme}100`,
       },
     },
 
     ':active': {
```
Comment (noeliacabane) [inline packages/native-ui/src/config/theme/helpers/Button.ts:27]: is `:active` pressed?

File: packages/native-ui/src/config/theme/helpers/Button.ts (hunk 1 above)
Comment (jordmccord) [reply packages/native-ui/src/config/theme/helpers/Button.ts:50]: Cheers, missed that one

File: packages/native-ui/src/config/theme/helpers/Button.ts (hunk 2 above)
Comment (jordmccord) [reply packages/native-ui/src/config/theme/helpers/Button.ts:27]: Yes 🙂

File: packages/native-ui/src/config/theme/helpers/Button.ts [hunk 3]
```diff
@@ -8,50 +8,53 @@ export const invertedOutlineScheme = (scheme: Scheme) => ({
...
       backgroundColor: `$${scheme}900`,
+      borderColor: `$${scheme}400`,
```
Comment (noeliacabane) [inline packages/native-ui/src/config/theme/helpers/Button.ts:29]: not sure if this applies to inverted only, 
on `inverted` false (light mode only) the border colour doesn't change on pressed (active) so in the case of cyan it stays at 400 and it the other `colorScheme stays` at 500 
Screenshot 2024-08-08 at 16 25 20

Comment (jordmccord) [reply packages/native-ui/src/config/theme/helpers/Button.ts:29]: Yes this is inverted only

Comment (jordmccord) [reply packages/native-ui/src/config/theme/helpers/Button.ts]: The normal outline did need updating though, good spot

**Review by noeliacabane — APPROVED** [review]
This looks good. Thanks so much

**Pull request merged**